num/src/bigint.rs

```diff
@@ -29,6 +29,12 @@ macro_rules! construct_bigint {
         pub struct $name([u64; $n_words]);
 
         impl $name {
+            #[inline]
+            /// Constructs an object from internal representation
+            pub const fn new(internal: [u64; $n_words]) -> $name {
```

Comment:
It is recommended to use `from_inner` for the function names constructing newtype from the inner representation
```suggestion
            pub const fn from_inner(inner: [u64; $n_words]) -> $name {
```

Reply:
fixed